sql/informix-sql/InformixSQLLexer.g4

```diff
@@ -257,5 +273,8 @@ SPACES: [ \u000B\t\r\n] -> channel(HIDDEN);
 
 UNEXPECTED_CHAR: .;
 
+fragment NEWLINE_EOF    : NEWLINE | EOF;
 fragment HEX_DIGIT: [0-9A-F];
 fragment DIGIT:     [0-9];
+fragment NEWLINE        : '\r'? '\n';
```

Comment:
On old Macs, a new line is a '\r'. It really should be added.

Reply:
But ANTLR runtime doesn't support `\r` line separators and is not going to support (ANTLR assigns incorrect lines and columns to recognized tokens): https://github.com/antlr/antlr4/issues/1098.

That's why I'd say adding `'\r'` as line separator doesn't make sense unless ANTLR supports it.